src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -221,20 +225,23 @@ public void periodic() {
                 m_hasAppliedOperatorPerspective = true;
               });
     }
-  }
 
-  public Command seedFieldCentric() {
-    return runOnce(io::seedFieldCentric);
+    modeEstimator.updateWithTime(
+        inputs.timestamp, inputs.gyroYaw, inputs.drivePositions, inputs.steerPositions);
   }
 
   public void resetPose(Pose2d pose) {
-    io.resetPose(pose);
+    modeEstimator.resetPose(pose);
   }
 
   /** Returns the current odometry pose. */
   @AutoLogOutput(key = "Odometry/Robot")
   public Pose2d getPose() {
-    return inputs.pose;
+    if (Constants.currentMode == Mode.REPLAY) {
+      return modeEstimator.getPose();
+    } else {
```

Comment:
you don't nee dthe else.

Reply:
it's probably optimized out anyway.. however..

Reply:
Change